src/controllers/projectController.js

```diff
@@ -0,0 +1,34 @@
+import { Router } from 'express';
+import HttpStatus from 'http-status-codes';
+import * as projectService from '../services/projectService';
+
+const router = Router();
+
+/**
+ * GET /api/admins
+ */
+router.get('/', (req, res, next) => {
+  projectService
+    .getRelatedProject(req.headers)
```

Comment:
Services should be reusable anywhere in the project (even from cli). Don't tie it with headers directly.
Instead, pass the params required the function directly.